components/match2/wikis/splitgate/match_summary.lua

```diff
@@ -0,0 +1,212 @@
+---
+-- @Liquipedia
+-- wiki=splitgate
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local DisplayHelper = require('Module:MatchGroup/Display/Helper')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local VodLink = require('Module:VodLink')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+
+local _EPOCH_TIME = '1970-01-01 00:00:00'
+local _EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+local htmlCreate = mw.html.create
```

Comment:
The variable is only used about half the time when creating a html object. Either drop the variable and always use the full name, or always use the variable